App/components/recommend.js

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from "react";
+import {
+  Alert,
+  Keyboard,
+  FlatList,
+  StyleSheet,
+  Text,
+  TouchableWithoutFeedback,
+  TouchableOpacity,
+  View,
+  ScrollView,
+} from "react-native";
+import DataBaseComponent from "../components/DatabaseComponent.js";
+import HistoryDataBase from "../../Screens/History.js";
+import Items from "../components/Items.js";
+import * as SQLite from "expo-sqlite";
+import { useEffect } from "react";
+import MyListItems from "./MyListItems.js";
+import { exp } from "react-native/Libraries/Animated/src/Easing";
+
+const db = SQLite.openDatabase("Grocery_Items.db");
+
+export default function Recommend({ submitHandler }) {
+  const [groceryItems, setGroceryItems] = useState([
+    { id: 0, name: "", expirationDate: "", price: "", addIt: false },
+  ]);
+
+  //const pressHandler = (id) => {};
+
+  useEffect(function effectFunction() {
+    update();
+  }, []);
+
+  //retrieving everything from the database, then putting them into an array and storing it in the state
+  update = () => {
+    db.transaction((tx) => {
+      tx.executeSql(`select * from items;`, [], (_, { rows: { _array } }) =>
+        setGroceryItems(_array)
+      );
+    });
+  };
+
+  const recycleWhenPressed = () => {
+    groceryItems.map((groceryItems) => {
+      if (groceryItems.name != null && !isNaN(groceryItems.price)) {
+        submitHandler(
+          groceryItems.name,
+          groceryItems.price,
+          "to be determined"
+        );
+      }
+    });
+  };
+
+  // const recycleWhenPressed2 = () => {
```

Comment:
We can probably delete this if it is not being used.

Reply:
Correct! I will delete it right now.